elements/rh-spinner/rh-spinner.ts

```diff
@@ -1,9 +1,10 @@
-import { LitElement, html } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { html } from 'lit';
 
 import { colorContextConsumer, colorContextProvider } from '../../lib/context/color.js';
 import type { ColorPalette, ColorTheme } from '../../lib/context/color.js';
 
+import { BaseSpinner } from '@patternfly/pfe-spinner/BaseSpinner';
```

Comment:
Nice! one thing though...

```suggestion
import { BaseSpinner } from '@patternfly/pfe-spinner/BaseSpinner.js';
```

We'll also have to double check that that file is listed in the exports (at least, until we combine the packages together)

Reply:
This change prevents the tests from working. I get this:

```
Browser logs:
      TypeError: Failed to fetch dynamically imported module: http://localhost:8001/elements/rh-spinner/test/rh-spinner.spec.ts?wtr-session-id=V9qsTQYOheFcHu1fZGT8s

 🚧 404 network requests:
    - node_modules/@patternfly/pfe-spinner/BaseSpinner.js.js

 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 
```

Reply:
this needs a fix in pfe-spinner then, to add BaseSpinner to the package.json exports field